packages/wix-ui-tpa/src/components/Dialog/Dialog.tsx

```diff
@@ -6,6 +6,7 @@ import { IconButton } from '../IconButton';
 import { ReactComponent as CloseIcon } from '../../assets/icons/Close.svg';
 import { TPAComponentProps } from '../../types';
 import { st, classes } from './Dialog.st.css';
+import ReactDOM = require('react-dom');
```

Comment:
The CI fails on:
`[npm run build --if-present] wix-ui-tpa: src/components/Dialog/Dialog.tsx(9,1): error TS1202: Import assignment cannot be used when targeting ECMAScript modules. Consider using 'import * as ns from "mod"', 'import {a} from "mod"', 'import d from "mod"', or another module format instead.`

I guess we should use ES modules syntax to import it:
```suggestion
import ReactDOM from 'react-dom';
```